.github/workflows/ci-build.yml

```diff
@@ -18,8 +18,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: volta-cli/action@v1
-        with:
-          node-version: 10.x
```

Comment:
Volta will use node 18. If we want to use an older version of node for some of the steps I can add these back w/ 14+.